inc/osvr/ClientKit/DisplayC.h

```diff
@@ -136,7 +173,8 @@ osvrClientCheckDisplayStartup(OSVR_DisplayConfig disp);
 
     @sa OSVR_ViewerCount
 
-    @return OSVR_RETURN_FAILURE if invalid parameters were passed, in which case
+    @return OSVR_RETURN_FAILURE if invalid parameters were passed, in
+   which case
```

Comment:
This is the main negative side-effect of clang format: weird comment breaking. Easiest fix is to merge everything into a single line and let it break the whole thing at once. In this case, since this code wasn't touched, committing this change was an "oops" on my part presumably when I went through and formatted.